modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -37,6 +37,19 @@
         verify_chain_of_trust    => true,
         verify_cot_signature     => true,
       },
+      'mobile-dep' => {
+        worker_group             => 'mobile-pushapk-dep-v1',
+        worker_type              => 'mobile-pushapk-dep-v1',
+        verbose_logging          => true,
+        taskcluster_client_id    => 'project/mobile/focus/releng/scriptworker/pushapk/dep',
```

Comment:
I wonder if we shouldn't have a different TC client here. Seems like `signing_scriptworker` has a similar definition of reusing the `focus` counterpart but I'm not sure why that is.

Reply:
Yeah, although `project:mobile:{product}` makes sense, client IDs like this one could be simplified to `project/mobile/releng/[...]`.

Let's keep it this way for now and add a comment to warn future readers.